pkg/handlers/oauth.go

```diff
@@ -70,13 +66,20 @@ func NewOAuthHandler(authConfig *config.ServiceAuthConfig) *OAuthHandler {
 	for _, clientCreds := range authConfig.Clients {
 		secretHashBytes, err := jwtauth.DecodeSecretHashFromConfig(clientCreds.SecretHash)
 		if err != nil {
-			log.Error().Err(err).Msgf("error decoding client creds secret (paired with client ID: %v)", clientCreds.ID)
+			log.Error().Err(err).Msgf("error decoding client creds secret (paired with client ID: %v), skipping these credentials", clientCreds.ID)
+			continue
+		}
+
+		if len(clientCreds.SDKKeys) == 0 {
+			log.Error().Err(err).Msgf("client creds missing or empty SDK keys (with client ID: %v), skipping these credentials", clientCreds.ID)
```

Comment:
Note: getting an access token for "Any SDK key" is not supported in this implementation. A specific set of SDK keys must be set, or the client will be skipped.

Reply:
Are there plans for global support?

Reply:
No. My intuition is that we shouldn't support it, but I am open to being persuaded otherwise.

Reply:
That's fine for now, we can add this option in the future.